demos/remote_attestation/dcap_rust/dcap_quote/c_app/dcap_quote.h

```diff
@@ -0,0 +1,30 @@
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "sgx_urts.h"
+#include "sgx_report.h"
+#include "sgx_qve_header.h"
+#include "sgx_dcap_ql_wrapper.h"
+#include "sgx_pce.h"
+#include "sgx_error.h"
+
+void *dcap_quote_open(void);
```

Comment:
Can we open the device automatically? Then both dcap_quote_open and dcap_quote_close are nor needed.